### PR TITLE
remove v3.10 deprecated maven/nodejs images

### PR DESCRIPTION
**Pull request by gabemontero**
@openshift/sig-developer-experience fyi

/assign @bparees

### CODE REVIEW

**Comment by gabemontero** [comment]
will most likely have to rebase the README in this one after the delete old plugin PR merges (assuming it merges first)

File: README.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ use with [OpenShift v3](https://github.com/openshift/origin)
 
 For an example of how to use it, [see this sample.](https://github.com/openshift/origin/blob/master/examples/jenkins/README.md)
 
-The images are pushed to DockerHub as openshift/jenkins-2-centos7, openshift/jenkins-slave-base-centos7, openshift/jenkins-slave-maven-centos7, openshift/jenkins-slave-nodejs-centos7, openshift/jenkins-agent-35-centos7, and openshift/jenkins-agent-nodejs-8-centos7.
+The images are pushed to DockerHub as openshift/jenkins-2-centos7, openshift/jenkins-slave-base-centos7, openshift/jenkins-agent-35-centos7, and openshift/jenkins-agent-nodejs-8-centos7.
```
Comment (grdryn) [inline README.md]: I think that should say `openshift/jenkins-agent-maven-35-centos7` instead of `openshift/jenkins-agent-35-centos7`.

Comment (gabemontero) [reply README.md]: thanks @grdryn .... update pushed

**Pull request merged**